documentation/ingestion_server/guides/migrate.md

```diff
@@ -0,0 +1,253 @@
+# Index migration runbook
+
+From time to time, we will need to update our Elasticsearch indices. These
+modifications can be classified into two broad-strokes categories, depending on
+whether the changes affect the main consumer of the indices, the API.
+
+## Migration types
+
+### API-free
+
+These changes are safe modifications to the ES schema that do not affect the
+API. As such they do not need any migration process. Examples:
+
+- addition of new fields or subfields
+- removal of fields that are not referenced or used by the API
+- changing the type to another compatible type (like `text` &harr; `keyword` )
+
+For API-free changes, we deploy the ingestion server and let a data-refresh to
+occur. The indexes will be updated to the new schema without manual intervention
+and will be made available to the API.
+
+### API-involved
+
+These changes are modifications to fields that already are in use by the API and
+involve code changes in both the ingestion server and the API. Examples:
+
+- removal of a field
+- changing the type to an incompatible type
+- renaming of a field
+
+Such kinds of changes need us to precisely deploy the API simultaneously with
```

Comment:
Since this doesn't _strictly_ have to happen simultaneously:
```suggestion
Such kinds of changes need us to precisely deploy the API in coordination with
```

Reply:
Implemented in 66134fa.